pulp_container/app/migrations/0020_update_push_repo_perms.py

```diff
@@ -0,0 +1,69 @@
+# Generated by Django 2.2.23 on 2021-06-01 14:53
+
+from django.contrib.auth.models import Group
+from django.db import migrations
+from guardian.shortcuts import assign_perm, get_objects_for_group
+
+
+def update_repo_perms(apps, schema_editor):
+    """ Assign push repo perms to the namespace and distribution groups. """
+    # Group = apps.get_model("auth", "Group")
```

Comment:
I get this traceback if I import Group through `get_model`
```
Groups = apps.get_model('auth', 'Group')
groups_ns1 = Groups.objects.filter(name__regex=r'container.(distribution|namespace).(collaborators|owners)')
type(groups_ns1[0])
Out[18]: __fake__.Group
get_objects_for_group(groups_ns1[0], 'container.modify_content_containerpushrepository', accept_global_perms=False)
Traceback (most recent call last):
  File "/usr/local/lib/pulp/lib64/python3.8/site-packages/IPython/core/interactiveshell.py", line 3437, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-19-6344fd0770a3>", line 1, in <module>
    get_objects_for_group(groups_ns1[0], 'container.modify_content_containerpushrepository', accept_global_perms=False)
  File "/usr/local/lib/pulp/lib64/python3.8/site-packages/guardian/shortcuts.py", line 766, in get_objects_for_group
    groups_obj_perms_queryset = (group_model.objects
  File "/usr/local/lib/pulp/lib64/python3.8/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/lib/pulp/lib64/python3.8/site-packages/django/db/models/query.py", line 892, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/usr/local/lib/pulp/lib64/python3.8/site-packages/django/db/models/query.py", line 910, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/pulp/lib64/python3.8/site-packages/django/db/models/sql/query.py", line 1290, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/pulp/lib64/python3.8/site-packages/django/db/models/sql/query.py", line 1315, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/usr/local/lib/pulp/lib64/python3.8/site-packages/django/db/models/sql/query.py", line 1224, in build_filter
    self.check_related_objects(join_info.final_field, value, join_info.opts)
  File "/usr/local/lib/pulp/lib64/python3.8/site-packages/django/db/models/sql/query.py", line 1084, in check_related_objects
    self.check_query_object_type(value, opts, field)
  File "/usr/local/lib/pulp/lib64/python3.8/site-packages/django/db/models/sql/query.py", line 1065, in check_query_object_type
    raise ValueError(
ValueError: Cannot query "Group object (1)": Must be "Group" instance.
```

Reply:
`get_objects_for_group` is probably not using the correct `__fake__`models.

Reply:
yeah, i plan to report this to django-guardian.